smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptWriter.java

```diff
@@ -266,7 +266,7 @@ boolean writeMemberDocs(Model model, MemberShape member) {
                     docs = docs.replace("{", "\\{")
                         .replace("}", "\\}");
                     if (member.getTrait(DeprecatedTrait.class).isPresent() || isTargetDeprecated(model, member)) {
-                        docs = "@deprecated\n\n" + docs;
+                        docs = docs + "\n\n@deprecated";
```

Comment:
this is needed since the tsdoc does not support explicit `@description` block like jsdoc. tsdoc uses only the implied description block and it must come first in the docblock order.